addons/web/static/src/views/fields/datetime/datetime_field.js

```diff
@@ -8,6 +8,7 @@ import {
     formatDateTime,
     today,
 } from "@web/core/l10n/dates";
+import { evaluateBooleanExpr } from "@web/core/py_js/py";
```

Comment:
fix conflict and adapt test(s) to hoot

Reply:
Helli @Polymorphe57 ,

I have made the changes, Could you please take a look at it?

Thank you🙂